.github/scripts/check_newsfragments.py

```diff
@@ -50,19 +55,12 @@ def check_newsfragment(fragment: Path, base: str) -> Optional[bool]:
             print(
                 f"[{fragment_name}] fragment ID doesn't correspond to an issue ! (On <{url}> HTTP error {exc.code} {exc.reason})"
             )
+            return False
         else:
+            # Sanity check is we can deserialize the response.
             data = json.loads(response.read())
             print(f"[{fragment_name}] issue#{id} => {data}")
-            if "pull_request" in data:
-                print(
-                    f"[{fragment_name}] fragment ID correspond to a pull request instead of an issue !"
-                )
-            else:
-                if data["state"] == "open":
-                    return True
-                else:
-                    print(f"[{fragment_name}] fragment ID correspond to a closed issue !")
-        return False
```

Comment:
No need to check if it's PR or closed issue?

Reply:
That check is problematic when you do a backport that include a newsfragment for an issue.

> That linked issue is likely already closed by the first PR that targeted the main branch.

IMO, I don't find that check relevant: 

1. Here we want to document an issue that as been resolve and link it in the changelog so it as nothing to do if the issue is open or not.
2. If later on we want to document additional issues that as been close in the changelog, that's not possible with that check.